tests/js/sentry-test/utils.tsx

```diff
@@ -27,3 +27,23 @@ export function textWithMarkupMatcher(textMatch: string | RegExp) {
     return findTextWithMarkup(element, textMatch);
   };
 }
+
+/**
+ * Trigger onPress events on components that use react-aria (e.g. the dropdown
+ * menu). These components require more complex events than what enzyme can
+ * simulate with `.simulate('click')`. Preferably, we should use the
+ * 'user-event' library from react testing library. This function is a
+ * workaround for test files that still use enzyme. Read more:
+ * https://react-spectrum.adobe.com/react-spectrum/testing.html#triggering-events
+ */
+export function triggerPress(element) {
```

Comment:
is there a type we can use for this parameter?

Reply:
I tried to type it but it caused a lot of issues in `selectDropdownMenuItem` 😛, since the `ReactWrapper` type isn't guaranteed to have an `onClick` event. The only way I know to fix that is to add type assertions everywhere in `selectDropdownMenuItem`, which seems rather onerous and unnecessary (people would only use these functions for elements that they know can be clicked).
<img width="772" alt="Screen Shot 2022-02-04 at 11 36 35 AM" src="https://user-images.githubusercontent.com/44172267/152592193-dec6ea5d-ab71-471a-bcc9-f49bbcd8bc36.png">
<img width="1005" alt="Screen Shot 2022-02-04 at 11 44 32 AM" src="https://user-images.githubusercontent.com/44172267/152593202-e6ff0f6b-d839-4292-a2e4-135c99c4fec3.png">